### DISCUSSION TITLE
spectral library formats

### DISCUSSION THREAD

**Discussion by rednaSander** [MannLabs/alphadia#375] [Questions / Support]
Hi, 

Issue:
The software crashes when attempting to load a .spectronaut.tsv spectral library file. 

CopyEnvironment:
- Data format: TIMS diaPASEF (.d)
- Library format: .spectronaut.tsv

Questions:
1. Is the .spectronaut.tsv format currently supported?
2. For the 'generic' .tsv library format mentioned in the documentation, could you please specify the required column headers and their expected contents?

Additional Information:
- The crash occurs during the library loading step
- No error message is displayed before the crash

**Comment by GeorgWa** [comment]
Hi, 
I moved the question here as it might be a bug #376.
Best,
Georg
